projects/graph/src/graph.py

```diff
@@ -2,9 +2,34 @@
 Simple graph implementation
 """
 
-
 class Graph:
     """Represent a graph as a dictionary of vertices mapping labels to edges."""
     def __init__(self):
         pass  # TODO
- 
\ No newline at end of file
+        self.vertices = {}
+    
+    def add_vertex(self, value):
+        if not value in self.vertices:
+            self.vertices[value] = set()
+    
+    def add_edge(self, vertex, target):
+        # print(self.vertices)
+        # print(self.vertices[vertex])
+        # print(hasattr(self.vertices, vertex))
+        if vertex in self.vertices and target in self.vertices:
+            self.vertices[vertex].add(target)
+        else:
+            print('One of the provided vertices does not exist.')
```

Comment:
well or both, but try to add some conditional logic that will reference the specif vertex(ices)

Reply:
Thank you! That has been updated.